recursion/Basic Questions/Factorial.py

```diff
@@ -0,0 +1,7 @@
+def factorial(n):
+    if n == 0:
+        return 1
```

Comment:
please add comments inside all files for better understanding

Reply:
ok